### PR TITLE
Travis CI support

### PR DESCRIPTION
**Pull request by hypery2k**
* basic build against iOS 9.3 and iOS 10 SDK
* Activate Travis CI here: https://travis-ci.org

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+sudo: false
+language: objective-c
+
+env:
+  - SDK="iphoneos9.3"
+  - SDK="iphoneos10.0"
+
+licenses:
+  - 'android-sdk-preview-license-52d11cd2'
+  - 'android-sdk-license-.+'
```
Comment (devxoul) [inline .travis.yml]: Why do we needs these lines?

**Comment by hypery2k** [comment]
you're right ;)
The basic build is working so far: https://travis-ci.org/hypery2k/Toaster/builds

**Comment by devxoul** [comment]
Thanks again, @hypery2k :+1::smile:

**Comment by hypery2k** [comment]
thanks to you for creating and maintaining this Pod ;)

**Pull request merged**